en/asgardeo/docs/tutorials/smart-on-fhir.md

```diff
@@ -0,0 +1,256 @@
+    <thead>
+        <th>API Resource</th>
+        <th>Authorized Scopes</th>
+    </thead>
+    <tbody>
+        <tr>
+            <td>OAuth DCR API</td>
+            <td>
+                <ul>
+                    <li>Create DCR</li>
+                    <li>Delete DCR</li>
+                    <li>View DCR</li>
+                    <li>Update DCR</li>  
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>SCIM2 Users API</td>
+            <td>
+                <ul>
+                    <li>Create User</li>
+                    <li>List Users</li>
+                    <li>View User</li>
+                    <li>Delete User</li>
+                    <li>Update User</li>
+                </ul>
+        </tr>
+        <tr>
+            <td>Application Management API</td>
+            <td>
+                <ul>
+                    <li>Create Application</li>
+                    <li>Delete Application</li>
+                    <li>Update Application</li>
+                    <li>View Application</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>Shared Application Management API</td>
+            <td>
+                <ul>
+                    <li>Create Shared Application</li>
+                    <li>View Shared Application</li>
+                    <li>Delete Shared Application</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>OIDC Scope Management API</td>
+            <td>
+                <ul>
+                    <li>Update OIDC Scopes</li>
+                    <li>Delete OIDC Scopes</li>
+                    <li>Create OIDC Scopes</li>
+                    <li>View OIDC Scopes</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>Claim Management API</td>
+            <td>
+                <ul>
+                    <li>Delete Claim</li>
+                    <li>Update Claim</li>
+                    <li>Create Claim</li>
+                    <li>View Claim</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>API Resource Management API</td>
+            <td>
+                <ul>
+                    <li>Delete API Resource</li>
+                    <li>Update API Resource</li>
+                    <li>Create API Resource</li>
+                    <li>View API Resource</li>
+                </ul>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+To do so,
+
+1. On the [Asgardeo Console](https://console.asgardeo.io/), go to **Applications**.
+
+2. Select the created application and go to its **API Authorization** tab.
+
+3. Do the following for all the REST APIs listed on the above table.
+
+    1. Click on **Authorize an API Resource** and do the following:
+    
+    2. Select the resource from the **API Resource** dropdown.
+
+        !!! note
+
+            Be sure to select the API resources listed under **Management API**.
```

Comment:
This note is not needed. The example was provided to enable the scopes for 'API Resource Management API’.

Reply:
Will reword